projects/step-core/src/lib/client/generated/services/PrivateApplicationService.ts

```diff
@@ -42,7 +42,7 @@ export class PrivateApplicationService {
   public getWebPlugins(): Observable<Array<AbstractWebPlugin>> {
     return this.httpRequest.request({
       method: 'GET',
-      url: '/app/plugins',
+      url: 'app/plugins',
```

Comment:
It is a generate service. It shouldn't be changed.

I guess change should be applied in `client/generated/core/OpenAPI` where rest base url is configured

Reply:
You are correct, I should have marked this as draft since we are still using it to test some issues.